.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+/logs/app.log
```

Comment:
it would be  better to exclude the whole logs folder, so if you will add a new logger file in the future, you won't need to add changes here